modules/misc/src/postprocessors/InternalVolume.C

```diff
@@ -22,7 +23,7 @@ validParams<InternalVolume>()
       "component", 0, "component<3", "The component to use in the integration");
   params.addParam<Real>(
       "scale_factor", 1, "A scale factor to be applied to the internal volume calculation");
-  params.addParam<Real>(
+  params.addParam<FunctionName>(
```

Comment:
Modify the doco so that it says time-dependent function is expected.

Reply:
I want the capability to supply a time-dependent function but want to also retain the capability of people supplying just a constant value that is constant in time as is in some of the existing tests.  

I wouldn't say that time-dependent function is expected but more of an option. I can add a second sentence that states "Can vary in time." or something along those lines.

Reply:
Yes. But, in general functions are time and space dependent. You are expecting just time-dependency - not spacial dependency. We cannot check if the supplied function is time-dependent, so when people supply space-dependent function, they will get constant behavior for point (0, 0, 0). So we need to capture that in the doco.